packages/admin/admin/src/mui/menu/ItemAnchorLink.tsx

```diff
@@ -1,9 +1,40 @@
+import { LinkExternal } from "@comet/admin-icons";
 import { ListItemButtonProps } from "@mui/material";
 import * as React from "react";
 
 import { MenuItem, MenuItemProps } from "./Item";
 
 export type MenuItemAnchorLinkProps = MenuItemProps & ListItemButtonProps & React.HTMLProps<HTMLAnchorElement>;
 
-// @ts-expect-error "component"-property is used as described in the documentation  https://mui.com/material-ui/react-list/, but type is missing in ListItemButtonProps
-export const MenuItemAnchorLink: React.FC<MenuItemAnchorLinkProps> = (props) => <MenuItem selected={false} component="a" {...props} />;
+export const MenuItemAnchorLink: React.FC<MenuItemAnchorLinkProps> = ({ secondaryAction, isMenuOpen, slotProps, ...props }) => {
+    const computedSecondaryAction =
+        secondaryAction !== undefined ? (
+            secondaryAction
+        ) : (
+            <>
+                <LinkExternal color="primary" sx={{ marginRight: "auto", marginLeft: 2 }} fontSize={isMenuOpen ? "medium" : "small"} />
+                {props.level}
+            </>
+        );
```

Comment:
nit: Usually we handle these cases like this:

```suggestion
export const MenuItemAnchorLink: React.FC<MenuItemAnchorLinkProps> = ({ secondaryAction: passedSecondaryAction, isMenuOpen, slotProps, ...props }) => {
    const secondaryAction =
        passedSecondaryAction ?? (
            <>
                <LinkExternal color="primary" sx={{ marginRight: "auto", marginLeft: 2 }} fontSize={isMenuOpen ? "medium" : "small"} />
                {props.level}
            </>
        );
```

(Suggestion is not correctly formatted)

Reply:
https://github.com/vivid-planet/comet/pull/2431/commits/20e012273cd283747308dfff430669535788bb7b